docs/build_docs.sh

```diff
@@ -89,6 +89,12 @@ if [[ "$4" != "" ]]; then
 else
     geoipsdocpath="$GEOIPS_PACKAGES_DIR/geoips/docs"
 fi
+if [[ "$5" != "" ]]; then
+    echo "passed geoips_vers=$5"
+    geoips_vers=$5
+else
+    geoips_vers=latest
+fi
```

Comment:
Note - defaulting to "latest" unless version is passed in command line.  When we get to the point of building docs for an actual version release, we will have to update some build_docs.sh call to actually pass in the version that we want.

Reply:
Makes sense. Thanks for the note.

Reply:
I'll leave comments open for the second reviewer to resolve.